augur/tasks/github/pull_requests/tasks.py

```diff
@@ -85,14 +85,14 @@ def process_pull_requests(pull_requests, task_name, repo_id, logger):
 
         # insert contributors from these prs
         logger.info(f"{task_name}: Inserting {len(contributors)} contributors")
-        session.insert_data(contributors, Contributor, ["cntrb_login"])
+        session.insert_data(contributors, Contributor, ["cntrb_id"])
 
 
         # insert the prs into the pull_requests table. 
         # pr_urls are gloablly unique across github so we are using it to determine whether a pull_request we collected is already in the table
         # specified in pr_return_columns is the columns of data we want returned. This data will return in this form; {"pr_url": url, "pull_request_id": id}
         logger.info(f"{task_name}: Inserting prs of length: {len(pr_dicts)}")
-        pr_natural_keys = ["repo_id", "pr_src_id"]
+        pr_natural_keys = ["pr_url"]
```

Comment:
@sgoggins / @IsaacMilarky This should be ["repo_id", "pr_src_id"] not ["pr_url"]. If a repo moves, the pr URL will change and we will get duplicate pr_src_ids since the pr src id stays the same, but the URL changes.

Reply:
I agree. I thought we changed this one already. ?